ui/lib/api.ts

```
@@ -0,0 +1,228 @@
+    this.client = axios.create({
+      baseURL: `http://localhost:${port}`,
+      headers: {
+        "Content-Type": "application/json",
+      },
+    });
+  }
+
+  private getErrorMessage(error: unknown): string {
+    if (isAxiosError(error) && error.response) {
+      const errorData = error.response.data as ErrorResponse;
+      if (errorData.error && errorData.error.message) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)